src/cargo/core/compiler/build_context/mod.rs

```diff
@@ -129,12 +129,12 @@ impl<'a, 'cfg> BuildContext<'a, 'cfg> {
         info.cfg().unwrap_or(&[])
     }
 
-    /// The host arch triple
+    /// Gets the host architecture triple.
     ///
-    /// e.g. x86_64-unknown-linux-gnu, would be
-    ///  - machine: x86_64
-    ///  - hardware-platform: unknown
-    ///  - operating system: linux-gnu
+    /// E.g., x86_64-unknown-linux-gnu, would be
```

Comment:
If we're going to add a comma, I'd say replace with "For example,".

Reply:
I'm not too bothered about this -- mainly following the Rustc codebase convention.

Reply:
I think "E.g.," is weird looking (personally) and I've seen wisdom in the past along the lines of "avoid Latin (like "e.g.") in favour of English (like "for example")".

Reply:
It is a bit... I know what you mean. I also like to avoid Latin abbreviations when English has a nearly-as-short-but-more-readable equivalent. That said, I was just trying to follow what I saw predominantly in the Rustc codebase.

Reply:
@dwijnand That's fair enough.